spec/services/configuration_respository_service_spec.rb

```diff
@@ -41,6 +41,7 @@
 
       expect(service).to receive(:create_remote_json_file).with("users.json", any_args)
       expect(service).to receive(:create_remote_json_file).with("projects.json")
+      expect(service).to receive(:create_remote_json_file).with("environment_variables.json")
```

Comment:
I want to add tests once I also cover https://github.com/fastlane/ci/issues/643 (the project/branch specific ENV variables)